src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Figure: ")
+                .append(" the side is of length: ")
+                .append(side)
+                .append(" and the color is ")
+                .append(color);
+        System.out.println(builder.toString());
```

Comment:
The `draw` method is conceptually different from getting the area of the figure. It's better to separate these responsibilities into different interfaces or classes.